styles/pronouns.cc/catppuccin.user.css

```
@@ -0,0 +1,216 @@
+    --bs-tertiary-bg: @surface0;
+    --bs-tertiary-bg-rgb: #rgbify(@surface0) [];
+    --bs-primary-text-emphasis: @accent-color;
+    --bs-secondary-text-emphasis: @subtext1;
+    --bs-success-text-emphasis: @green;
+    --bs-info-text-emphasis: @teal;
+    --bs-warning-text-emphasis: @yellow;
+    --bs-danger-text-emphasis: @red;
+    --bs-light-text-emphasis: @subtext0;
+    --bs-primary-bg-subtle: @mantle;
+    --bs-secondary-bg-subtle: @base;
+    --bs-primary-border-subtle: @blue;
+    --bs-secondary-border-subtle: @overlay0;
+    --bs-success-border-subtle: @green;
+    --bs-info-border-subtle: @teal;
+    --bs-warning-border-subtle: @yellow;
+    --bs-danger-border-subtle: @red;
+    --bs-light-border-subtle: @overlay1;
+    --bs-dark-border-subtle: @surface1;
+    --bs-heading-color: inherit;
+    --bs-link-color: @blue;
+    --bs-link-hover-color: @blue;
+
+    --bs-code-color: @pink;
+    --bs-highlight-color: @rosewater;
+    .list-group {
+      --bs-list-group-color: var(--bs-body-color);
+      --bs-list-group-bg: var(--bs-body-bg);
+      --bs-list-group-border-color: var(--bs-border-color);
+      --bs-list-group-border-width: var(--bs-border-width);
+      --bs-list-group-border-radius: var(--bs-border-radius);
+      --bs-list-group-item-padding-x: 1rem;
+      --bs-list-group-item-padding-y: 0.5rem;
+      --bs-list-group-action-color: var(--bs-secondary-color);
+      --bs-list-group-action-hover-color: var(--bs-emphasis-color);
+      --bs-list-group-action-hover-bg: var(--bs-tertiary-bg);
+      --bs-list-group-action-active-color: var(--bs-body-color);
+      --bs-list-group-action-active-bg: var(--bs-secondary-bg);
+      --bs-list-group-disabled-color: var(--bs-secondary-color);
+      --bs-list-group-disabled-bg: var(--bs-body-bg);
+      --bs-list-group-active-color: @base;
+      --bs-list-group-active-bg: @accent-color;
+      --bs-list-group-active-border-color: @accent-color;
+      border-radius: var(--bs-list-group-border-radius);
+    }
+    // navbar
+    .navbar {
+      --bs-navbar-color: fade(@text, 55%) !important;
+      --bs-navbar-hover-color: fade(@text, 75%) !important;
+      --bs-navbar-disabled-color: fade(@text, 25%) !important;
+      --bs-navbar-active-color: @text !important;
+      --bs-navbar-brand-color: @text !important;
+      --bs-navbar-brand-hover-color: @text !important;
+      --bs-navbar-toggler-border-color: fade(@crust, 10%) !important;
+    }
+    #coloredButton(@color, @bg,@hover) {
+      --bs-btn-color: @color;
+      --bs-btn-bg: @bg;
+      --bs-btn-border-color: @bg;
+      --bs-btn-hover-color: @hover;
+      --bs-btn-hover-bg: @bg;
+      --bs-btn-hover-border-color: @bg;
+      --bs-btn-focus-shadow-rgb: #rgbify(@bg) [];
+      --bs-btn-active-color: @hover;
+      --bs-btn-active-bg: @bg;
+      --bs-btn-active-border-color: @bg;
+      --bs-btn-disabled-color: @text;
+      --bs-btn-disabled-bg: @bg;
+      --bs-btn-disabled-border-color: @bg;
+    }
```

```suggestion
    #coloredButton(@color, @bg) {
      --bs-btn-color: @color;
      --bs-btn-bg: @bg;
      --bs-btn-border-color: @bg;
      --bs-btn-hover-color: @color;
      --bs-btn-hover-bg: darken(@bg, 5%);
      --bs-btn-hover-border-color: darken(@bg, 5%);
      --bs-btn-focus-shadow-rgb: #rgbify(@bg) [];
      --bs-btn-active-color: @color;
      --bs-btn-active-bg: darken(@bg, 5%);
      --bs-btn-active-border-color: darken(@bg, 5%);
      --bs-btn-disabled-color: @color;
      --bs-btn-disabled-bg: @bg;
      --bs-btn-disabled-border-color: @bg;
    }
```
Update the rest of the buttons below to remove the third argument, I don't think it is necessary.